src/table/context/TableContext.tsx

```diff
@@ -1,16 +1,29 @@
 import React, { useState, useReducer, createContext } from 'react';
+import { stardust } from '@nebula.js/stardust';
 import PropTypes from 'prop-types';
 import { createSelectorProvider } from './createSelectorProvider';
 import { reducer } from '../utils/selections-utils';
 
-export const TableContext = createContext();
+export const TableContext = createContext({});
 
 const ProviderWithSelector = createSelectorProvider(TableContext);
 
-export const TableContextProvider = ({ children, selectionsAPI, cellCoordMock, selectionDispatchMock }) => {
+interface ContextProviderProps {
+  children: JSX.Element;
+  selectionsAPI: stardust.ObjectSelections;
+  cellCoordMock: [number, number];
+  selectionDispatchMock: jest.Mock<any, any>;
```

Comment:
any is used often. Shall we disable `Unexpected any. Specify a different type.eslint[@typescript-eslint/no-explicit-any](https://typescript-eslint.io/rules/no-explicit-any)`?

Reply:
Hmm not sure... This is the type that comes from jest so we can't type more than this. But in other cases it might just be laziness and then there should be a warning. If anything it can be disabled for this specific line maybe. Let's keep it in mind and see what we want to do when we have converted some more